README.md

```diff
@@ -1,3 +1,19 @@
+# Major Update!
```

Comment:
One thing that will be slightly awkward is that the `README.md` file will be used on both pub and on GitHub.  On pub we want this message explaining that the package is discontinued, but that will be confusing for the GitHub repository.

We will need to commit this, publish to pub.dev, and then immediately commit a change to the `README.md` that reverts it.

Reply:
I was thinking about publishing to pub.dev without committing. This change, therefore, will be reverted in this PR.

The update is only for pub's benefit, as you point out, it doesn't need to live on this repo (as part of changelog too).

Reply:
https://pub.dev/packages/flutter_widgets update completed (along with minor version bump and changelog entry) and I reverted the README.md change here.

Reply:
Ok. (Somehow your earlier explanation about publishing without committing didn't sink in.)